src/RuleSet/Sets/PERCS1x0Set.php

```diff
@@ -39,6 +42,10 @@ public function getRules(): array
 
     public function getDescription(): string
     {
-        return 'Rules that follow `PER Coding Style 1.0 <https://www.php-fig.org/per/coding-style/>`_.';
+        return <<<'DESC'
+            **This ruleset is deprecated** in favour of `@PER-CS2.0`.
```

Comment:
TODO: create `DeprecatedRuleSetDescriptionInterface`, similar to `DeprecatedFixerInterface`

Reply:
=> https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7285